elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -2,7 +2,7 @@
 
 $LOCAL: cta;
 
-$pfe-cta__arrow-size: 0.313em;
```

Comment:
I don't see this referenced anywhere else? Was it actually in use?

Reply:
same, i saw no references to it and nothing complained when i removed it, so i think its old cruft